Data/ByteString.hs

```diff
@@ -1858,7 +1862,7 @@ mkPS buf start end =
 
 mkBigPS :: Int -> [ByteString] -> IO ByteString
 mkBigPS _ [ps] = return ps
-mkBigPS _ pss = return $! concat (P.reverse pss)
+mkBigPS _ pss  = return $! concat (P.reverse pss)
```

Comment:
Let's keep this line unchanged, please. I often dig through `git blame` and would prefer to keep collateral damage minimal ;)

Reply:
ah sorry I had lint on save on, did all the imports and everything else!! thought I'd taken them all out but I missed this! 😅